dashboard/test/factories.rb

```diff
@@ -393,6 +393,7 @@
         evaluator.section.try(:user) ||
         build(:teacher)
       follower.section = evaluator.section || build(:section, user: follower.user)
+      follower.user_id = follower.section.user_id
```

Comment:
If the one is just an alias for the other, I don't quite understand why this extra attribute-setter in the factory is necessary to establish their equality. Which of the tests fail without this added line?